administrator-manual/en/backup.rst

```diff
@@ -186,6 +186,17 @@ Supported storage backends:
 * Restic `REST server <https://github.com/restic/rest-server>`_
 
 
+When configuring a backup using the restic engine and a remote storage backend,
+please ensure you have enough bandwidth to complete the first backup within
+24 hours. Otherwise restic will create multiple different snapshots.
+If you have a slow connection and you still want to use a remote storage
+backend, follow this procedure:
+
+- configure the restic backup
+- manually execute the backup by clicking on :guilabel:`Run now`
+- disable the configured backup, so it will not start at next scheduled execution
+- when the backup is over, re-enable it to allow scheduled execution
+
```

Comment:
```suggestion

When configuring a backup using the Restic engine and a remote storage backend,
ensure you have enough bandwidth to complete the first backup within
24 hours. Otherwise Restic will create multiple different snapshots.
If you have a slow connection and you still want to use a remote storage
backend, follow this procedure:

- configure the Restic backup
- manually execute the backup by clicking on :guilabel:`Run now`
- disable the configured backup, so it will not start at next scheduled execution
- when the backup is over, re-enable it to allow scheduled execution
```